toolchain/check/merge.h

```diff
@@ -56,6 +56,32 @@ auto ReplacePrevInstForMerge(Context& context, SemIR::NameScopeId scope_id,
                              SemIR::NameId name_id, SemIR::InstId new_inst_id)
     -> void;
 
+// Common information shared by different kinds of named entity, such as classes
+// and functions.
+struct EntityInfo {
```

Comment:
"EntityInfo" sounds like a generic name for something that's only used by merge. Do you expect additional fields will be added so that it can be used elsewhere? Is this header the right home? Maybe something like `DeclParamInfo` might be more descriptive of the use-case?

Reply:
Renamed to `DeclParams`. We can generalize later if needed.